.github/workflows/release.yml

```diff
@@ -0,0 +1,326 @@
+        echo "CERT_NAME=${{secrets.AZ_CERT_NAME}}" | Out-File -FilePath $env:GITHUB_ENV -Append
+        echo "VAULT_ID=${{secrets.AZ_VAULT_ID}}" | Out-File -FilePath $env:GITHUB_ENV -Append
+        echo "APP_ID=${{secrets.AZ_APP_ID}}" | Out-File -FilePath $env:GITHUB_ENV -Append
+        echo "TENANT_ID=${{secrets.AZ_TENANT_ID}}" | Out-File -FilePath $env:GITHUB_ENV -Append
+        echo "CLIENT_SECRET=${{secrets.AZ_CLIENT_SECRET}}" | Out-File -FilePath $env:GITHUB_ENV -Append
+    - name: Pandoc Setup
+      uses: r-lib/actions/setup-pandoc@v2
+      with:
+        pandoc-version: '3.1.11'
+    # Note: Windows assigns a UUID to the installer at build time, it's assumed
+    # by windows that one release version == one UUID (always).  Breaking this assumption
+    # has some rather nasty side-effects in windows, such as possibly breaking 'uninstall'
+    # functionality. We should avoid clobbering or re-building windows installers in most cases,
+    # For build-only, the .exe is saved in the workflow artifacts for a human
+    # to judge.
+    - name: Build
+      id: build
+      run: |
+        Push-Location contrib\win-installer
+        .\build.ps1 ${{steps.getversion.outputs.version}} prod ${{ github.workspace }}\release-artifacts
+        $code = $LASTEXITCODE
+        if ($code -eq 2) {
+          Write-Output "artifact-missing=true" | Out-File -FilePath $env:GITHUB_OUTPUT -Append
+          Pop-Location
+          Exit 0
+        }
+        Pop-Location
+        Exit $code
+    - name: Display structure of downloaded files
+      run: |
+        Push-Location contrib\win-installer
+        Get-ChildItem
+        Pop-Location
+    - name: Artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: win-installer
+        path: |
+          .\contrib\win-installer\podman-${{steps.getversion.outputs.version}}-setup.exe
+
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    if: needs.check.outputs.buildonly == 'false'
+    needs: [check, build-artifacts, mac-pkg, windows-installer]
+    permissions:
+      contents: write
+    env:
+      VERSION: ${{needs.check.outputs.version}}
+    steps:
+    - name: Checkout Version
+      uses: actions/checkout@v4
+      with:
+        ref: ${{needs.check.outputs.version}}
+    - name: Get release notes
+      run: |
+        ver="$(echo "$VERSION" | sed -e "s/^v//" -e "s/-rc.*//")"
+        releasenotes="$VERSION-release-notes.md"
+        awk -v ver=$ver '/^## / { if (p) { exit }; if ($2 == ver) { p=1; next } } p' RELEASE_NOTES.md > $releasenotes
+        if [[ -z $(grep '[^[:space:]]' $releasenotes) ]]; then
+          if [[ $VERSION != *-rc* ]]; then
+              echo "::notice:: Release does not have release notes"
+              exit 1
+          else
+          echo "This is a release candidate of Podman v$ver. Full release notes will be available with a later RC." > $releasenotes
+          fi
+        fi
+    - name: Get release notes
```

Comment:
duplicate name "Get release notes"

Reply:
just two minor things then I can slash-lgtm it

Reply:
Updated, thanks!